libraries/apollo-mockserver/src/commonMain/kotlin/com/apollographql/apollo3/mockserver/MockServer.kt

```diff
@@ -235,11 +244,20 @@ internal class MockServerImpl(
 }
 
 @JsName("createMockServer")
-fun MockServer(): MockServer = MockServerImpl(QueueMockServerHandler(), true, TcpServer(), null)
+fun MockServer(port: Int = 0): MockServer = MockServerImpl(
+    QueueMockServerHandler(),
+    true,
+    TcpServer(port),
+    null)
 
 @Deprecated("Use MockServer.Builder() instead", level = DeprecationLevel.ERROR)
 @ApolloDeprecatedSince(ApolloDeprecatedSince.Version.v4_0_0)
-fun MockServer(handler: MockServerHandler): MockServer = MockServerImpl(handler, true, TcpServer(), null)
+fun MockServer(handler: MockServerHandler, port: Int = 0): MockServer =
```

Comment:
I think this change may not be binary compatible - to be sure, can you run `./gradlew apiDump` and add the changes to the PR. This will give a report of the changes to the public API, so we can check everything is additive only.

Reply:
Given it's deprecated, I would even not touch this function at all. Or is the `Builder` too cumbersome @sdfgsdfgd ? (In which case we should probably not deprecated it)

Reply:
@martinbonnin  just to be sure, given the changes approved, are you expecting any changes here (it's probably due for removal anyways, I guess I was just keeping the code consistent across the board 🙂 )

Reply:
I tweaked things a bit in [this commit](https://github.com/apollographql/apollo-kotlin/pull/5389/commits/556eb62a48274206fdece28fdb274095a76b6c2b), will merge once the build passes